### PR TITLE
added changelog info

### PR DESCRIPTION
**Pull request by jvanasco**
Referencing Mike's comment in : https://github.com/sqlalchemy/sqlalchemy-access/pull/3#issuecomment-582032243

I've Integrated the copy I proposed in : https://github.com/sqlalchemyorg/sqlalchemyorg/pull/1

### CODE REVIEW

File: develop.html.mako [hunk 1]
```diff
@@ -112,6 +113,26 @@ requested changes are not responded towards will be closed.</li>
 <li>Please make sure you are configured to run the current <a href="#environment">Development Environment</a>
     requirements, which will help ensure your PRs pass automatic tests.</li>
 
+<li>Instead of editing the Changelog file itself, please adopt the following workflow:
+    <ul>
+        <li>Create a per-issue changelog file with the path
+            `/doc/build/changelog/unreleased_{Version}/{IssueId}.rst`, in which
+            `IssueId` is the auto-generated GitHub Issue Id.
+            </li>
+        <li>If the PR does not have a corresponding issue, use a placeholder file
```
Comment (zzzeek) [inline develop.html.mako]: any PR that warrants a changelog entry would need to have an issue number.    I very rarely make changelog-worthy changes without creating an issue and outside contributors shouldn't be doing it ever.

File: develop.html.mako [hunk 2]
```diff
@@ -93,7 +93,8 @@ It is a frequent occurence that pull requests submitted either don't describe
 what problem they intend to solve, or they solve the problem in an inappropriate
 or sub-optimal way.   Having a bug or feature issue opened ahead of time
 will allow us to to understand the use case and discuss the best approach
-before the contributor makes the effort to write the code and tests.</li>
+before the contributor makes the effort to write the code and tests.  The Github
```
Comment (zzzeek) [inline develop.html.mako]: "The issue number used by Github is also linked in the changelog and is used to generate a changelog file entry for merge."

File: develop.html.mako [hunk 3]
```diff
@@ -112,6 +113,26 @@ requested changes are not responded towards will be closed.</li>
...
+    <ul>
+        <li>Create a per-issue changelog file with the path
+            `/doc/build/changelog/unreleased_{Version}/{IssueId}.rst`, in which
+            `IssueId` is the auto-generated GitHub Issue Id.
+            </li>
+        <li>If the PR does not have a corresponding issue, use a placeholder file
+            in your PR to generate a unique identifier for the PR, and then rename
+            the placeholder file to use the PR identifer for the `IssueId`.
+            </li>
+        <li>The changelog file format will hopefully be self-documenting
```
Comment (zzzeek) [inline develop.html.mako]: "The file format for a changelog file uses the .rst markup described at https://github.com/sqlalchemyorg/changelog .   See the README.txt file in the doc/build/unreleased_XX" directory for an example"

(then we have to add examples of the format in the README.txt files.... )

**Comment by CaselIT** [comment]
@jvanasco any update on this one?

**Comment by jvanasco** [comment]
Thanks for the ping. I think I forgot to push the commits? Will do tomorrow!

**Comment by jvanasco** [comment]
abandoning this in favor of #5

**Pull request closed without merging**